structuralcodes/sections/_generic.py

```diff
@@ -604,25 +606,49 @@ def _rotate_triangulated_data(self, theta: float):
         self.triangulated_data = rotated_triangulated_data
 
     def integrate_strain_profile(
-        self, strain: ArrayLike
-    ) -> t.Tuple[float, float, float]:
+        self,
+        strain: ArrayLike,
+        integrate: t.Literal['stress', 'modulus'] = 'stress',
+    ) -> t.Union[t.Tuple[float, float, float], NDArray]:
         """Integrate a strain profile returning internal forces.
```

Comment:
Suggest to change to: ...returning internal forces or section stiffness.

Reply:
sure, thanks!